nixos/modules/misc/version.nix

```diff
@@ -5,11 +5,12 @@ with lib;
 let
   cfg = config.system;
 
-  releaseFile  = "${toString pkgs.path}/.version";
-  suffixFile   = "${toString pkgs.path}/.version-suffix";
-  revisionFile = "${toString pkgs.path}/.git-revision";
-  gitRepo      = "${toString pkgs.path}/.git";
-  gitCommitId  = lib.substring 0 7 (commitIdFromGitRepo gitRepo);
+  releaseFile     = "${toString pkgs.path}/.version";
+  suffixFile      = "${toString pkgs.path}/.version-suffix";
+  nixosState      = import "${toString pkgs.path}/.version-state.nix";
+  revisionFile    = "${toString pkgs.path}/.git-revision";
+  gitRepo         = "${toString pkgs.path}/.git";
+  gitCommitId     = lib.substring 0 7 (commitIdFromGitRepo gitRepo);
```

Comment:
Nitpicking, but I think the whitespace changes here before `=` aren't necessary.